resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverChannelStrategy.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.resolver.dns;
+
+/**
+ * Strategy that influence how {@link io.netty.channel.socket.DatagramChannel}s are used during queries.
+ */
+public enum DnsNameResolverChannelStrategy {
+    /**
+     * Use the same underlying {@link io.netty.channel.socket.DatagramChannel} for all queries.
+     */
+    Same,
+    /**
+     * Use a new {@link io.netty.channel.socket.DatagramChannel} per resolve chain or per explicit query.
```

Comment:
Should we clarify that in this case behavior will be similar to JDK resolver and it will cause more overhead?